.github/workflows/genesis-verify.yml

```diff
@@ -0,0 +1,57 @@
+name: genesis-verify
+
+on:
+  pull_request:
+    branches:
+      - master
+    paths:
+      - "misc/deployments/**/genesis.json"
+
+jobs:
+  verify:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v41
+        with:
+          files: "misc/deployments/**/genesis.json"
+
+      - name: List changed genesis files
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
+            echo "Changed file: $file"
+          done
```

Comment:
Why do you need this?

Reply:
Oh it's for debugging purpose for now, it should be removed when pr goes into ready for review status

Reply:
52bcdf81353d9c44d85f10fa871ca6ad7f4ac6ff